### PR TITLE
Trigger fixes

### LINKED ISSUE #167: triggers in osc are broken
They no longer work - I didn't go back to find out who broke this or when.

### PR DESCRIPTION
**Pull request by dNechita**
This fixes #167

### CODE REVIEW

File: osc.c [hunk 1]
```diff
@@ -1230,7 +1230,7 @@ static gboolean capture_process(void *data)
 			if (offset / (off_t)sizeof(gfloat) < info->offset / 4) {
 				offset = 0;
 			} else if (offset) {
-				offset -= info->offset * sizeof(gfloat) / 4;
+				offset -= (info->offset / 4) * sizeof(gfloat);
 				for (i = 0; i < nb_channels; i++) {
```
Comment (commodo) [inline osc.c]: the commit could use a bit more description about some of the intricate details of this fix

Comment (commodo) [reply osc.c]: i'm referring to commit `osc: Fix trigger corrupting all data when capturing an odd nb of samples`
Github reports them in a weird way

**Comment by commodo** [comment]
rest looks good

File: osc.c (hunk 1 above)
Comment (dNechita) [reply osc.c]: Done

**Pull request merged**